src/node_util.cc

```diff
@@ -161,6 +161,12 @@ static void SetHiddenValue(const FunctionCallbackInfo<Value>& args) {
   args.GetReturnValue().Set(maybe_value.FromJust());
 }
 
+static void Sleep(const FunctionCallbackInfo<Value>& args) {
```

Comment:
This should have an `env->PrintSyncTrace()` call, imo

Reply:
I agree with that.